rotkehlchen/chain/evm/decoding/thegraph/decoder.py

```diff
@@ -58,6 +81,73 @@ def _decode_delegator_staking(self, context: DecoderContext) -> DecodingOutput:
             return self._decode_stake_locked(context)
         elif context.tx_log.topics[0] == TOPIC_STAKE_DELEGATED_WITHDRAWN:
             return self._decode_stake_withdrawn(context)
+        elif context.tx_log.topics[0] == DELEGATION_TRANSFERRED_TO_L2:
+            return self._decode_delegation_transferred_to_l2(context)
+
+        return DEFAULT_DECODING_OUTPUT
+
+    def _get_user_address(self, address: ChecksumEvmAddress) -> ChecksumEvmAddress | None:
+        """Get the user address (benficiary) from the vesting contract if it exists"""
+        if self.evm_inquirer.get_code(address) != '0x':
+            try:
+                raw_beneficiary_address = self.base.evm_inquirer.call_contract(
+                    contract_address=address,  # the vesting contract
+                    abi=GRAPH_TOKEN_LOCK_WALLET_BENEFICIARY_ABI,
+                    method_name='beneficiary',
+                )
+                return deserialize_evm_address(raw_beneficiary_address)
+            except RemoteError as e:
+                log.error(f'During delegation transfer to arbitrum got error calling {address} beneficiary(): {e!s}')  # noqa: E501
+                return None
+        return address
+
+    def _decode_delegation_transferred_to_l2(self, context: DecoderContext) -> DecodingOutput:
+        """Decode a transfer delegating GRT from ethereum to arbitrum"""
+        user_address = self._get_user_address(hex_or_bytes_to_address(context.tx_log.topics[1]))
+        if not user_address or not self.base.is_tracked(user_address):
+            return DEFAULT_DECODING_OUTPUT
```

Comment:
newline after return

```suggestion

            return DEFAULT_DECODING_OUTPUT
```